mr diff: /home/elmig/debian/debian-installer/. (in subdir /home/elmig/debian/debian-installer/manual)
Index: build/entities/urls.ent
===================================================================
--- build/entities/urls.ent	(revision 67004)
+++ build/entities/urls.ent	(working copy)
@@ -46,6 +46,10 @@
 
 <!ENTITY email-debian-boot-list "debian-boot@lists.debian.org">
 
+<!-- support -->
+<!ENTITY url-debian-user-forums "http://forums.debian.net/">
+<!ENTITY url-ask-debian-net "http://ask.debian.net/">
+
 <!-- base for the current distribution -->
 <!ENTITY disturlftp "&url-debian-ftp;debian/dists/&releasename;/">
 <!ENTITY disturl    "&url-archive;debian/dists/&releasename;/">
Index: en/boot-new/boot-new.xml
===================================================================
--- en/boot-new/boot-new.xml	(revision 67004)
+++ en/boot-new/boot-new.xml	(working copy)
@@ -33,9 +33,11 @@
 
 If you are new to &debian; and &arch-kernel;, you may need some help from more
 experienced users.
-<phrase arch="x86">For direct on-line help you can try the IRC channels
-#debian or #debian-boot on the OFTC network. Alternatively you can contact
-the <ulink url="&url-list-subscribe;">debian-user mailing list</ulink>.</phrase>
+<phrase arch="x86">For direct on-line help you can try the IRC channel
+#debian on the OFTC network. Alternatively you can contact
+the <ulink url="&url-list-subscribe;">debian-user mailing list</ulink>, 
+<ulink url="&url-debian-user-forums;">Debian User Forums</ulink> or even 
+<ulink url="&url-ask-debian-net;">ask.debian.net</ulink>.</phrase>
 <phrase arch="not-x86">For less common architectures like &arch-title;,
 your best option is to ask on the
 <ulink url="&url-list-subscribe;">debian-&arch-listname; mailing

mr diff: finished (1 ok)
